### ISSUE TITLE
apiserver.enabled mentioned in docs, but not referenced in code?

### ISSUE THREAD

**Issue by keeganwitt** [appscode/kubed#400]
The latest install.md (https://github.com/appscode/kubed/blob/aa438f38d8e8db202f2ddc97122df9fd0f16a817/docs/setup/install.md#using-helm) has `--set apiserver.enabled=false`, yet I don't see this referenced in any of the chart template files.  Is this still a valid option?  If not, can you explain the background and motivation behind the change?

Looking at the docs on Helm hub, it looks like it was present in [v0.11.0](https://hub.helm.sh/charts/appscode/kubed/v0.11.0), but removed in [v0.12.0-rc.0](https://hub.helm.sh/charts/appscode/kubed/v0.12.0-rc.0).

The flag is also no longer mentioned on the website (https://appscode.com/products/kubed/v0.12.0/setup/install/).

**Comment by martaver** [comment]
Yeah this is quite annoying for me... all I want is to sync a secret across namespaces and here I am trying to configure certificates for an apiserver that I'm not even using, in order to prevent terraform/pulumi from detecting changes every time i run a plan, because the certificate is regenerated each time the chart is deployed.
